style/_tabs.scss

```diff
@@ -48,7 +48,7 @@
     cursor: pointer;
     font-size: var(--sd-fontsize-tabs-label);
     font-weight: 700;
-    padding: 1em 1.25em 0.5em;
+    padding: 0.5em 1.25em 0.5em;
```

Comment:
As mentioned in the PR, this makes the top and bottom padding symmetrical. It's unclear to me why there was more top padding to begin with. I thought maybe it was to create some white space above the tabbed panel, but it already has top margin, so this value is a bit puzzling to me.

I may be missing something here.